clang/www/cxx_dr_status.html

```diff
@@ -10698,7 +10698,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
     <td><a href="https://cplusplus.github.io/CWG/issues/1815.html">1815</a></td>
     <td>CD4</td>
     <td>Lifetime extension in aggregate initialization</td>
-    <td class="unreleased" align="center">Clang 19</td>
+    <td class="full" align="center">Yes</td>
```

Comment:
Unrelated changes snuck in.

Reply:
This particular one is an unfortunate oversight from #87933, and will be fixed after #92113 is merged.
Other unrelated changes are by design since #90352, when we started to fetch CWG index from source that updates much more often that official publication.

Reply:
It's usually better to apply the unrelated changes to the tree first, then base the patch on top of that; this way reviewers don't see unrelated changes in the patch.

Reply:
I updated the branch from `main`, now with #92113 changes. Unrelated changes are gone.